newsletter-may-2023.rst

```diff
@@ -0,0 +1,87 @@
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+   :category: Newsletter
+
+Read the Docs newsletter - May 2023
+===================================
+
+News and updates
+----------------
+
+- 🚁️ The proxy application El Proxito has been rewritten and the new version is rolled out to selected projects.
+  The application resolves URLs for all documentation websites hosted on Read the Docs.
+  The new rewrite improves the performance of the resolver and makes it possible to add planned features.
+  We are implementing these changes gradually,
+  while monitoring their stability.
+- 🔎️ ...One of the new features that have been added,
+  is an improved 404 page,
+  which contains better error messages and tips for users and project owners.
+  It is available to projects using the new Proxito implementation.
+- 💫️ We now support multiple ``.readthedocs.yaml`` files in the same repository.
+  This is especially useful for *monorepos* containing multiple documentation projects with different configurations.
+  This allows for instance configurations of several projects with different documentation tools and build environments.
+  Read more about the feature :doc:`in our docs <readthedocs:guides/setup/monorepo>`.
+- ⚙️ If you use ``build.commands`` in ``.readthedocs.yaml``,
+  you are no longer required to have a ``build.tools`` section.
+  We changed the validation for ``.readthedocs.yaml`` to accommodate projects that do not need any of the built-in tools exposed.
+- 🐛️ Fixed: URLs on pull request builds were pointing to the build page,
+  rather than the documentation preview.
+  If a build is successful,
+  the URL now points to the documentation preview.
+- 🐛️ Fixed: `An issue <https://github.com/readthedocs/readthedocs.org/issues/10290#issuecomment-1542841524>`__ in our legacy build images caused builds using Sphinx to start failing on May 4th when urllib3 2.0.2 was released.
+  The issue has been fixed by upgrading OpenSSL on these images.
+
+  We still encourage to specify a newer build image,
+  an example is given `in this comment <https://github.com/readthedocs/readthedocs.org/issues/10290#issuecomment-1535120995>`__.
+- 🔒️ Vulnerability fixed: `CAS session hijacking on Read the Docs for Business <https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-4mgr-vrh5-hj8q>`__
+
+Upcoming features
+-----------------
+
+- 🚁️ As mentioned earlier,
+  we have rewritten our proxy application El Proxito and will start to add new features.
+  One of the features being worked on is the ability to serve :doc:`subprojects <readthedocs:subprojects>` at custom URL paths.
+
+.. figure:: img/screenshot-404-juxtaposed.png
+   :alt: A screenshot of the old and new versions of our 404 pages.
+   
+   Left: the classic old 404 page with the ascii art maze - right: new contextualized 404 pages.
+   We are slowly rolling out the new default 404 pages that are meant to be more helpful to users and project owners.
```

Comment:
I think this image should go where we talk about the 404 improvements instead here, that we are talking about subprojects at custom URL paths